Dockerfile

```diff
@@ -359,6 +359,21 @@ RUN \
 # https://gitlab.alpinelinux.org/alpine/aports/-/issues/11806
 RUN sed -i 's/-lgcc_s//' /usr/local/lib/pkgconfig/rav1e.pc
 
+# bump: librtmp /LIBRTMP_COMMIT=([[:xdigit:]]+)/ gitrefs:https://git.ffmpeg.org/rtmpdump.git|re:#^refs/heads/master$#|@commit
+# bump: librtmp after ./hashupdate Dockerfile LIBRTMP $LATEST
+# bump: librtmp link "Commit diff $CURRENT..$LATEST" https://git.ffmpeg.org/gitweb/rtmpdump.git/commitdiff/$LATEST?ds=sidebyside
+ARG LIBRTMP_URL="https://git.ffmpeg.org/rtmpdump.git"
+ARG LIBRTMP_COMMIT=f1b83c10d8beb43fcc70a6e88cf4325499f25857
+RUN \
+  git clone "$LIBRTMP_URL" && \
+  cd rtmpdump && git checkout $LIBRTMP_COMMIT && \
+  # Patch/port librtmp to openssl 1.1
+  for _dlp in dh.h handshake.h hashswf.c; do \
+    wget $WGET_OPTS https://raw.githubusercontent.com/microsoft/vcpkg/master/ports/librtmp/${_dlp%.*}.patch; \
```

Comment:
Could we have a commit hash instead of master in the URL to make things stabler?

Reply:
maybe have a patch folder within the project?

Reply:
i dont have much time but... you could also have a patcjed forked on your name, since librtmp hasnt been updated in ages

Reply:
feel free to use this code and create your own with it